tools/cli/go-whisk/whisk/activation.go

```diff
@@ -43,6 +47,8 @@ type Activation struct {
     Logs            []string    `json:"logs"`
     Annotations     KeyValueArr `json:"annotations"`
     Publish         *bool       `json:"publish,omitempty"`
+    BufferSize      int         `json:",omitempty"`
```

Comment:
since this struct is a representation of the activation json returned from the controller. we probably should avoid adding fields that are not returned in the json object.  it would be nice to separate out this extra data into a parent struct.